hocr-lines

```diff
@@ -20,4 +20,4 @@ doc = html.fromstring(stream.read())
 lines = doc.xpath("//*[@class='ocr_line']")
 
 for line in lines:
-    print get_text(line)
+    print get_text(line).encode("utf-8")
```

Comment:
It makes sense to look also at the other occurrences of `get_text` https://github.com/tmbdev/hocr-tools/search?utf8=%E2%9C%93&q=get_text (search again is maybe better)